pennylane/fermi/fermionic.py

```diff
@@ -123,7 +123,7 @@ def __str__(self):
 
     def __repr__(self):
         r"""Terminal representation of a FermiWord"""
-        return str(self)
+        return str(dict(self))
```

Comment:
```suggestion
        return str(self.sorted_dic)
```
maybe this is better?

Reply:
I committed this change.